AutomatedTesting/Gem/PythonTests/AWS/Windows/aws_metrics/aws_metrics_automation_test.py

```diff
@@ -3,24 +3,38 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
 
 SPDX-License-Identifier: Apache-2.0 OR MIT
+
+
+Prerequisites for running AWS automation tests on your local Windows machine:
+1) Go to the AWS IAM console and create an IAM role called o3de-automation-tests which adds your own AWS account as
+   a trusted entity and has the admin permissions
+2) Copy o3de\scripts\build\Platform\Windows\deploy_cdk_applications.cmd to your engine root folder
+3) Open a Command Prompt window and set the following environment variables:
+   Set O3DE_AWS_PROJECT_NAME=AWSAUTO
+   Set O3DE_AWS_DEPLOY_REGION=us-west-2

```

Comment:
nit: Is this enforced? Does it have to be us-west-2? Is the developer able to choose? It would be better to always use US standard as the default unless theres a requirement to run out of us-west-2.

Reply:
When running these tests on Jenkins, we need to make sure that we're running these tests at the same region as the one Jenkins job deploy all the CDK applications.